docs/data/scheduler/timezone-support/TimezoneDataset.tsx

```diff
@@ -13,42 +12,20 @@ import {
 export default function TimezoneDataset() {
   const eventModelStructure: SchedulerEventModelStructure<TimezoneEvent> = {
     start: {
-      getter: (event) => {
-        const d = parseISO(event.start);
-
-        return new TZDate(
-          d.getFullYear(),
-          d.getMonth(),
-          d.getDate(),
-          d.getHours(),
-          d.getMinutes(),
-          d.getSeconds(),
-          d.getMilliseconds(),
-          event.timezone,
-        );
-      },
+      getter: (event) => new TZDate(event.startUtc, event.timezone),
       setter: (event, newValue) => {
-        event.start = format(newValue, "yyyy-MM-dd'T'HH:mm:ss");
+        const tz = (newValue as any).timeZone;
+        event.startUtc = newValue.toISOString();
```

Comment:
Can you add a small callout on the doc saying that this DX is temporary and we plan to support the `event.timezone` out of the box in the future?